content/en/docs/handling-errors.md

```diff
@@ -2,76 +2,157 @@
 title: "Handling Errors in the AWS SDK for Go V2"
 linkTitle: "Handling Errors"
 date: "2020-11-12"
-description: "Use the Error interface to handle errors from the |sdk-go| or AWS service."
+description: "Use the Error interface to handle errors from the AWS SDK for Go V2 or AWS service."
 ---
 
-The {{% alias sdk-go %}} returns errors that satisfy the Go `error`
-interface type and the `awserr.Error` interface in the
-``aws/awserr`` package. You can use the `Error()` method to get a formatted string of
-the SDK error message without any special handling.
+The {{% alias sdk-go %}} returns errors that satisfy the Go `error` interface type  You can use the `Error()` method to
+get a formatted string of the SDK error message without any special handling. Errors returned by the SDK may implement
+an `Unwrap` method. The `Unwrap` method is used by the SDK to provide additional contextual information to errors, while
+providing access to the underlying error or chain of errors. The `Unwrap` method should be used with the
+[errors.As](https://golang.org/pkg/errors#As) and [errors.Is](https://golang.org/pkg/errors#Is) functions which can
```

Comment:
We probably want to highlight errors.As, because error.Is matches value not compares against type. Generally errors.As will be more correct.